### DISCUSSION TITLE
Unable to read query/nx.log files on Ventura 13.2.1

### DISCUSSION THREAD

**Discussion by blackice19** [DNSCrypt/dnscrypt-proxy#2341] [Potential issues]
`query.log`, `nx.log` in v2.1.4; 

## *When* does this occur?

when .log is saved in a seperate folder location, the file gets created but when double clicking to open it, message displayed'unable to read file'. 

## *Where* does it happen?

v2.1.4 on OSX 13.2.1. issue not seen in dnsproxy-encrypt v 2.1.3 running on OSX 13.2.1

## *How* do we replicate the issue?

1. in OSX 13.2.1, set the path to save the .log fike in /Users folder path.
2. connect to the internet
3.  post connection, when u start surfing, the.log file is corrected created there. 
4. but on opening it , message displyed'unable to read file'
5. permissions to dnscrypt-proxy st to full disk access.
6. permission to .log file set to read and write, but no use

## Expected behavior (i.e. solution)

can open and read the file

## Other Comments

Is it expected behaviour (if so, how to read the file), a bug or am I missing something.

Thanks

**Comment by jedisct1** [comment]
This is due to a breaking change in a dependency:

https://github.com/natefinch/lumberjack/pull/83

Being restrictive by default isn't a bad idea, but the fact that it changes the default behavior is not great, especially since we have no ways to restore the previous behavior without downgrading.

https://github.com/natefinch/lumberjack/pull/168 solves this, but it still waiting to be merged.

**Reply by blackice19** [reply]
Hi Team,

Quick question please-when are we expecting this isssue to be fixed? Is there a workaround to view the log files. Thanks

**Reply by jedisct1** [reply]
That was worked around in 47e6a56b166a3b89e5d7dc1da7e0863494b24b14
